mlflow/pipelines/steps/ingest/__init__.py

```diff
@@ -89,7 +89,7 @@ def _run(self, output_directory: str) -> BaseCard:
 
         step_card = self._build_step_card(
             ingested_dataset_profile=ingested_dataset_profile,
-            ingested_rows=ingested_df.size,
```

Comment:
Any idea what the `size` actually returns?

Reply:
For a dataframe it's `rows` x `cols`

Reply:
Found this link useful: https://stackoverflow.com/questions/57691690/difference-between-len-and-size
It explains the difference and shows that size returns

Reply:
Makes sense now! Thanks!